ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -64,37 +64,48 @@ def check_dtl(logger):
 
     @staticmethod
     @timeout_decorator.timeout(15)
-    def _check_volumedriver(vp_name, test_name):
+    def _check_volumedriver(volume_name, vpool_guid, volume_size=10*1024**3):
```

Comment:
Wasted cpu cycles because its always the same calculation, we could create a STATIC variable here with the whole number in it and a comment next to it: this is 10GB